README.md

```diff
@@ -14,24 +14,24 @@ Before installing this plugin, you must ensure that `phpmd` is installed on your
 2. Install [pear](http://pear.php.net).
 
 3. Install `phpmd` by typing the following in a terminal:
-   ```
-   pear channel-discover pear.phpmd.org
-   pear channel-discover pear.pdepend.org
-   pear install --alldeps phpmd/PHP_PMD
-   ```
+ ```ShellSession
```

Comment:
`sh` and `bash` are aliases for `Shell`, which is meant for `Shell` scripts, since this is meant to be a series of commands the user types in I felt `ShellSession` was a better fit.

https://github.com/github/linguist/blob/master/lib/linguist/languages.yml